lib/wsdl.js

```diff
@@ -79,12 +79,12 @@ function trim(text) {
 }
 
 /**
- * What we want is to copy properties from one object to another one and avoid 
- * properties overriding. This way we ensure the 'inheritance' of 
+ * What we want is to copy properties from one object to another one and avoid
+ * properties overriding. This way we ensure the 'inheritance' of
```

Comment:
Could you please revert all of the changes which are not related to code but did change whitespaces and eol? we ask for this so we can keep our history clean and really just related to code changes where they belong.